stream-chat-android-ui-components-sample/src/main/kotlin/io/getstream/chat/ui/sample/feature/home/HomeFragment.kt

```diff
@@ -69,6 +73,18 @@ class HomeFragment : Fragment() {
         }
     }
 
+    private fun parseNotificationData() {
```

Comment:
Did you also check push notifications in the old sample app?

Reply:
Not yet, I'll review it

Reply:
Old sample app has a general problem with notifications not being delivered. I'll add the redirection feature once it's fixed. So let's merge those in a separate PRs.